docs/framework/index.md

```diff
@@ -0,0 +1,22 @@
+--
+title: .NET Framework
+description: .NET Framework
+keywords: .NET, .NET Server, .NET Desktop
+author: dotnet-bot
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net
+ms.technology: .net
+ms.devlang: dotnet
+ms.assetid: a27b2ae4-154e-4b2b-b221-0c4c05185274
+---
+
+# .NET Framework
+
+Under this section you'll find information about the .NET Framework
+on Windows. 
```

Comment:
Well, this is framework that only runs on Windows. How do we want to best distinguish this from .NET Core and .NET Standard?